app/controllers/devise/devise_authy_controller.rb

```diff
@@ -177,6 +180,19 @@ def request_sms
 
   private
 
+  def register_totp(mfa_config)
```

Comment:
Does this belong in here instead of the controller? https://github.com/MidwestAccessCoalition/twilio-verify-devise/blob/main/lib/twilio-verify-devise/client.rb

Reply:
Hmmm, I think this belongs here. Although I do notice that I need to rename the friendly_name on line 185